charts/dapr/charts/dapr_rbac/templates/priorityclassresourcequota.yaml

```
@@ -2,6 +2,7 @@
      this is required in some cases to ensure that the priority class is allowed in the namespace
      https://kubernetes.io/docs/concepts/policy/resource-quotas/#limit-priority-class-consumption-by-default */}}
 {{ if or (eq .Values.global.priorityClassName "system-node-critical") (eq .Values.global.priorityClassName "system-cluster-critical") }}
+{{- /* Deprecated. Will be remove on 1.16 */}}
```

I think this has to be removed in 1.17 if Im reading our [breaking change policy](https://docs.dapr.io/operations/support/breaking-changes-and-deprecations/#process-for-applying-breaking-changes) correct here. If it goes in this release: 1.15 then 2 releases from now would be 1.17.
```suggestion
{{- /* Deprecated. Will be remove on 1.17 */}}
```